Components/toast/toast.css

```diff
@@ -0,0 +1,55 @@
+@import url('../../styles/globalVar.css');
+@import url('../button/button.css');
+.toast-container{
+    flex-wrap: wrap;
+    margin: 1rem;
+    row-gap: 1rem;
+}
+
+.toast{
+    border-radius: 5px;
+    bottom: 1rem;
+    display: none;
+    left: 1rem;
+    padding: 0.625rem 2rem;
+    position: absolute;
+    position: absolute;
```

Comment:
You have written it twice.

Reply:
@manish591 Thanks for pointing it out, removed the redundant property